test/3.x/no_layout_app.js

```diff
@@ -27,16 +27,36 @@ app.use(express.static(__dirname + '/public'));
 
 app.get('/', function(req, res){
   res.render('no_layout', {
-    title: 'Express Handlebars Test',
+    title: 'Express Handlebars Test'
   });
 });
 
 app.get('/with_layout', function(req, res){
   res.render('blank', {
-    layout: 'layout'
+    layout: 'layout',
+    title: 'Express Handlebars Test'
```

Comment:
I added this test and then they where failing on the title and blank.hbs hade a linebreak that I removed, then they worked. Im not sure if i put the test in the right test suite but it seemed like the most equivalent.